CMakeLists.txt

```
@@ -155,16 +155,22 @@ if(FMIDI_PROGRAMS)
   target_link_libraries(fmidi-read PRIVATE fmidi fmidi-fmt)
   install(TARGETS fmidi-read
     RUNTIME DESTINATION "bin")
+  install(FILES "${PROJECT_SOURCE_DIR}/man/fmidi-read.1"
```

It is unnecessary to write a full path here.
```suggestion
  install(FILES "man/fmidi-read.1"
```